app/models/container_group.rb

```diff
@@ -69,6 +71,14 @@ def event_where_clause(assoc = :ems_events)
     end
   end
 
+  def tenant_identity
+    if ext_management_system
+      ext_management_system.tenant_identity
+    else
+      User.super_admin.tap { |u| u.current_group = Tenant.root_tenant.default_miq_group }
+    end
```

Comment:
I don't know the context but if it's on classes that include the TenancyMixin, would it make sense to go there?

https://github.com/ManageIQ/manageiq/blob/323d3ce875b87f9a581d17b33ef0e06d43103125/app/models/mixins/tenancy_mixin.rb#L1